packages/office-addin-sso/src/cli.ts

```diff
@@ -4,7 +4,7 @@
 // Licensed under the MIT license.
 
 import * as commander from "commander";
-import { logErrorMessage } from "office-addin-cli";
+import { logErrorMessage } from "office-addin-usage-data";
```

Comment:
package.json change?

Reply:
It is there as well. Probably already imported.

Reply:
But you can now remove cli from package.json

Reply:
This one still had a call for parseNumber